disk/tests/test_unit.py

```diff
@@ -155,3 +157,29 @@ def test_min_disk_size(aggregator, gauge_metrics, rate_metrics):
         aggregator.assert_metric_has_tag(name, 'device:{}'.format(DEFAULT_DEVICE_NAME))
 
     aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.skipif(sys.platform != 'linux', reason='disk labels are only available on Linux')
+@pytest.mark.usefixtures('psutil_mocks')
+def test_labels_from_blkid_cache_file(aggregator, instance_blkid_cache_file, gauge_metrics, rate_metrics):
+    """
+    Verify that the disk labels are set with when the blkid_cache_file option is set
+    """
+    c = Disk('disk', {}, [instance_blkid_cache_file])
+    c.check(instance_blkid_cache_file)
+    for metric in chain(gauge_metrics, rate_metrics):
+        aggregator.assert_metric(metric, tags=['device:/dev/sda1', 'label:MYLABEL'])
```

Comment:
One issue is that the `psutil_mocks` is only returning `/dev/sda1`. Since labels are tags that are eventually applied only to metrics that are discovered through the return values of `psutil_mocks`, adding a second device implies to modify that mock to be able to return multiple devices. I thought it was a bit out of scope for this PR, since it would probably imply to change a lot of other tests that depend on this mock too. However, if you feel like it's important, I can take a stab at it.